### PR TITLE
ci: run tests on Node.js v20 and v19

### PR DESCRIPTION
**Pull request by snitin315**
#### What changes did you make? (Give an overview)

- run tests on Node.js v20
- Remove Node v19 and v17 from matrix

#### Is there anything you'd like reviewers to focus on?
I feel it's fine to test on even node version only.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
+        node: [20.x, 18.x, 16.x, 14.x, 12.x, "12.22.0"]
```
Comment (mdjermanovic) [inline .github/workflows/ci.yml:30]: Can we keep 17 and 19, like in https://github.com/eslint/eslint/pull/17093?

(the same for other packages)

Comment (snitin315) [reply .github/workflows/ci.yml:30]: Yes, updating my PRs accordingly.

Comment (snitin315) [reply .github/workflows/ci.yml:30]:
```suggestion
        node: [20.x, 19.x 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
+        node: [20.x, 19.x 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
```
Comment (snitin315) [inline .github/workflows/ci.yml:30]:
```suggestion
        node: [20.x, 19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
```

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Review by Playaset — COMMENTED** [review]
import json from "./foo.json" with { type: "json" };
import("foo.json", { with: { type: "json" } });

**Pull request merged**